src/Integration.UnitTests/Binding/BindingWorkflowTests.cs

```diff
@@ -385,49 +384,12 @@ public void BindingWorkflow_InjectAdditionalFilesIntoProjects()
             testSubject.AdditionalFilePaths.Add(additionalFilePath1);
             testSubject.AdditionalFilePaths.Add(additionalFilePath2);
 
-            var msbuildProject1 = new MSBuild.Project();
-            var msbuildProject2 = new MSBuild.Project();
-            this.projectSystemHelper.MsBuildProjectMapping.Add(project1, msbuildProject1);
-            this.projectSystemHelper.MsBuildProjectMapping.Add(project2, msbuildProject2);
-
-            // Act
-            testSubject.InjectAdditionalFilesIntoProjects();
-
-            // Verify
-            VerifyAdditionalFileAddedToProject(project1, msbuildProject1, additionalFilePath1);
-            VerifyAdditionalFileAddedToProject(project2, msbuildProject2, additionalFilePath1);
-        }
-
-        [TestMethod]
-        public void BindingWorkflow_InjectAdditionalFilesIntoProjects_ExistingFile_DoesNotAddToProject()
```

Comment:
Not at this level. The `ProjectSystemHelper` is handling the check for already existing files whereas previously the InjectAdditionalFiles method was doing work to resolve whether or not to add the file.